control_panel_api/models.py

```diff
@@ -37,6 +37,10 @@ def aws_create_role(self):
     def aws_delete_role(self):
         services.delete_role(self.iam_role_name)
 
+    def helm_create_user(self):
```

Comment:
This is fine, you may omit the `user` bit as you're in the context of the `User` model, maybe rename to just `helm_create` or something like that.

Reply:
👍